models/image.py

```diff
@@ -545,14 +546,14 @@ def merge_all(self, session):
             self.sources.provenance_id = self.sources.provenance.id if self.sources.provenance is not None else None
             new_image.sources = self.sources.merge_all(session=session)
 
-            new_image.wcs = new_image.sources.wcs
-            if new_image.wcs is not None:
+            if new_image.sources.wcs is not None:
```

Comment:
Would there be any value in making image.wcs, image.wcs, image.zp all into `@property` things that point to image.sources.<thing> rather than having them be manually loaded like this?